projects/plugins/protect/src/js/components/firewall-header/index.jsx

```diff
@@ -105,10 +105,10 @@ const FirewallHeader = ( { status, hasRequiredPlan } ) => {
 								{ __( 'Active', 'jetpack-protect' ) }
 							</Text>
 							<H3 className={ styles[ 'firewall-heading' ] } mb={ 1 } mt={ 2 }>
-								{ hasRequiredPlan
+								{ automaticRulesApply && ! manualRulesApply
```

Comment:
```suggestion
								{ automaticRulesApply &&
```
This would consider any time automatic rules being enabled to output "Automatic firewall is on".